DOCS-8626-automatic-interval (#139)
mulesoft/docs-monitoring: modules/ROOT/pages/app-dashboards.adoc

The following charts appear on the *Overview* dashboard:

Inbound - Total Requests by Response Type::

> The sum of inbound requests, grouped by type of response by automatic interval. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.

Reviewer: Are the requests grouped by type or interval or both? Is it one sum but displayed by type or interval? Or is it all the OK ones in an interval, and all the Failed ones in an interval?
It might be easier to say 
```suggestion
The sum of all `OK` and all `Failed` inbound requests during an interval. If a Mule app has more than one connector type, all types are included in this metric.
```
I'm also unclear how the second sentence ties in to the first, mostly because we use "type" again, but in a different way.

Reviewer: I would like to hear more about this; I'm really confused by it.

Inbound - Average Response Time::

> The average response time of inbound requests, grouped by automatic interval. If a Mule app has more than one connector type, all types are included in this metric.

Reviewer: ditto: so, the average of each interval or the average of all intervals?
And the second sentence: what does connector type have to do automatic intervals?

Mule Messages::

> The total number of Mule messages that are processed successfully, grouped by automatic interval.

Reviewer: ditto

Maybe we should include a "For example, ..." in the first text like this, clarifying how the metric appears "grouped by automatic intervals."

Outbound - Total Requests by Response Type::
The sum of outbound requests, grouped by type of response by automatic interval. Responses are `OK` or `Failed`. If a Mule app has more than one connector type, all types are included in this metric.

Outbound - Average Response Time::
The average response time of outbound requests, grouped by automatic interval. If a Mule app has more than one connector type, all types are included in this metric.

JVM - CPU % Utilization::

> The average percentage of CPU used by the JVM, grouped by worker by automatic interval. The JVM used in this chart is the Mule JVM process. The calculation for CPU utilization is relative to the physical server on which Mule is deployed. For example, in RTF, the resource usage indicates the JVM use with respect to the entire server, and not with respect to the pod maximum CPU allocation.

Reviewer: GLOBALLY, can we replace utilization and utilized by "used"?
